src/main/resources/view/PersonListCard.fxml

```diff
@@ -5,10 +5,12 @@
 <?import javafx.scene.layout.ColumnConstraints?>
 <?import javafx.scene.layout.FlowPane?>
 <?import javafx.scene.layout.GridPane?>
+<?import javafx.scene.image.ImageView?>
 <?import javafx.scene.layout.HBox?>
 <?import javafx.scene.layout.Region?>
 <?import javafx.scene.layout.VBox?>
 
+<?import javafx.scene.image.ImageView?>
```

Comment:
A very small nit here: I think the import on line 13 might be redundant and the import one line 8 should be shifted before `javafx.scene.layout.*`.